### PR TITLE
NHS Dashboard Integration /dash

### PR DESCRIPTION
**Pull request by nathanhebe**
Initial integration via plugin of the NHS England Dashboard v1

### CODE REVIEW

File: ckanext/nhsengland/plugin.py [hunk 1]
```diff
@@ -119,6 +119,7 @@ def before_map(self, route_map):
         """
         with routes.mapper.SubMapper(route_map,
             controller='ckanext.nhsengland.plugin:NHSEController') as m:
+            m.connect('dashboard', '/dash', action='dashboard')
```
Comment (ntoll) [inline ckanext/nhsengland/plugin.py]: As mentioned in the email... how about we put this stuff under a `/dashboards` root..? So we get `/dashboards/board-report` as the second argument. Any subsequent reports can be `/dashboards/foo` etc etc etc

**Comment by ntoll** [comment]
From the CKAN/Python point of view this is exactly what we're looking for. If you're happy, then I'm happy. :-)

Since you said you were going to make further changes I'll wait to merge into our master branch until you give me the nod (as a comment on here I guess).

Cheers.

File: ckanext/nhsengland/plugin.py (hunk 1 above)
Comment (nathanhebe) [inline ckanext/nhsengland/plugin.py]: I initially tried m.connect('dashboard', '/dashboard', action='dashboard')

Comment (ntoll) [inline ckanext/nhsengland/plugin.py]: Hey hey,

Regarding the recursive loop. It turns out that CKAN already has something at the end of the `/dashboard` path and by adding it (again) in our plugin weird things happen, birds fly backwards, the sun sets in the north and zombies attack. ;-)

Comment (nathanhebe) [inline ckanext/nhsengland/plugin.py]: Aww no, so CKAN had the dashboard built in all this time - damn it :-)

**Comment by nathanhebe** [comment]
Hi ntoll,

**Comment by ntoll** [comment]
Hi,

So it's deployed (https://data.england.nhs.uk/dash) but I can't see anything. Obviously there's a bug. Happy to help in any way that I can.

N.

**Comment by ntoll** [comment]
OK... turns out that Firefox was blocking something. It looks great! :-)

**Comment by ntoll** [comment]
OK... looking around, I notice there are some missing graphical elements - e.g. I see a broken image on the finance page.

**Comment by ntoll** [comment]
OK... hard coded URL to other instance was causing the Firefox collywobbles. See the email David sent and my reply.

File: ckanext/nhsengland/plugin.py (hunk 1 above)
Comment (davidmiller) [inline ckanext/nhsengland/plugin.py]: This got merged before anyone actually fixed it.

Recursive loop now on the production site.

Sad faces.

Comment (nathanhebe) [inline ckanext/nhsengland/plugin.py]: What is the fix?

**Pull request merged**